services/workflows-service/prisma/migrations/20231218141329_add_workflow_definition_variant/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE "WorkflowDefinition" ADD COLUMN     "variant" TEXT NOT NULL DEFAULT 'DEFAULT';
```

Comment:
NOT NULL for existing env is a bit problematic because default wont take in place for existing rows in workflow definitions, so this migration will fail for our deployed environments

Reply:
I am not getting it.
so it will be "DEFAULT" until something else is set.

what is wrong here?